recipes/mentalist/0.1.9/meta.yaml

```diff
@@ -10,19 +10,19 @@ source:
   sha256: 44f6718c7d64d952e439fbddba6a413558c95b00de6f3daf0c0807b850914c3e
 
 build:
-  number: 2
+  number: 3
   skip: True  # [osx]
 
 requirements:
   host:
-    - julia
-    - hdf5
-    - mpfr
+    - julia 0.5.2
+    - hdf5 1.8.17
```

Comment:
This pinning is not needed anymore if you can use the global pinning that are in place. E.g this one: https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L343

If you think julia needs to be pinned, we should add julia as well to the above linked file.

Reply:
Sorry, I feel like I'm missing some understanding of how pinning is supposed to work under conda-build 3.

We have two versions of MentaLiST (a 0.1 series and a 0.2 series). The 0.1 series is written against `julia-0.5.2` and the 0.2 series is written against `julia-0.6.1`. So I need to ensure that the correct version of Julia is installed for each version of MentaLiST.

Reply:
Yes, but hdf5 is for example globally pinned. Julia 1.0 should probably be added as a global pin as well.

Reply:
Ah, I'm beginning to understand. Sorry, I feel like I've been a bit dense while adjusting to some of the new cb3 stuff. The docs are good, but it's taken me some time to absorb them.